docs-content/using.html.md.erb

```diff
@@ -110,26 +131,60 @@ In the command above:
 * There is a single free service plan called `community`.
 * For convenience, you may use the same instance-name in multiple PCF spaces, so that the same application manifest works in all spaces.
 
+**For Conjur Open Source and Conjur Enterprise v5:**
+Adding a service instance to a space will automatically create resources within Conjur
+to allow privileging all of the apps in that org or space to access a secret in Conjur. These
+resources are Conjur layers named using the Globally Unique Identifiers (GUIDs) for the Org and
+Space.
+
+The Org and Space GUIDs may be retrieved from CloudFoundry using the `cf` CLI tool. For example:
+```
+$ cf org --guid my-org
+8f366b77-d826-4522-8cc6-261bf9b02cba
+
+$ cf space --guid my-space
+4b02b69c-ba35-4069-b634-ded5644740c9
+```
+
+Assuming the Service Broker policy branch is `pcf/production`, the org and space layers
+may be used to acces variables in Conjur policy like:
+```
+   # Entitle all applications in my-org
+   - !grant
+     role: !group my-app/secrets-users
+     member: !layer /pcf/production/8f366b77-d826-4522-8cc6-261bf9b02cba
+
+   # Entitle all applications in my-space
+   - !grant
+     role: !group my-app/secrets-users
+     member: !layer /pcf/production/8f366b77-d826-4522-8cc6-261bf9b02cba/4b02b69c-ba35-4069-b634-ded5644740c9
+```
+
 #### <a id="edit-manifest"></a> Step 3: Edit Application Manifest
 
-Add the instance-name from the previous step to the list of services in the application manifest (if you are using a manifest). For example:
+Add the instance-name from the previous step to the list of services in the application manifest (if you are using a manifest). If you are using the Conjur Summon buildpack, it must also be specified in the manifest.
+
+For example:
 
 ```
 applications:
  - name: my-app
    services:
    - my-service-instance-name
+   buildpacks:
+   - conjur_buildpack
+   - ... other buildpacks
 ```
 
 #### <a id="push-app"></a> Step 4: Push Application to PCF Space
 
-If your application lists buildpacks in its manifest or you need to list buildpacks in the `cf push` command, see [Use a Custom BuildPack](#custom-bp) before proceeding with this step.
-
 Target your desired orgs and spaces and push the application and its associated files.
 
 <pre class='term'>cf push ...</pre>
 
-Use the `--no-start` option to avoid starting the application before it is configured to work with Conjur.
+If you are going to have Conjur secrets only accesible to this application, use the
+`--no-start` option to avoid starting the application before it is configured to work
+with Conjur.
 
```

Comment:
It's probably worth adding a notable note here that says something like

> If you're using Conjur EE v5+ or Conjur OSS v1+ and you're entitling your applications to access secrets in Conjur at the org or space level, then you're done!

Or maybe adding a head that is like
> Optional additional steps for Conjur EE v4.x or entitling individual applications